requirements.txt

```diff
@@ -70,7 +70,7 @@ responses==0.9.0
 
 # A common package that holds the models deifnition and schemas that are used
 # accross different amundsen repositories.
-amundsen-common>=0.3.1,<1.0
+amundsen-common==0.3.4
```

Comment:
Let's try to keep this as is for now, we don't want to fix any versions either that would still lead to problems the next time there's an amundsen-common upgrade. At least with the previous syntax I *think* its a matter of just running the right scripts to optionally upgrade vs having to change the code here to get the new version.